recipes-core/images/nilrt-restore-mode-image.bb

```diff
@@ -1,7 +1,7 @@
 DESCRIPTION = "Tiny initramfs image intended to run restore mode operations, uses minimal-nilrt-image"
-NIBOOT_BUNDLE_IMAGE = "minimal-nilrt-bundle"
-IMAGE_DISPLAY_NAME = "minimal runmode"
+NIBOOT_BUNDLE_IMAGE = "nilrt-base-bundle"
```

Comment:
Well, we don't really have anything more minimal at the moment. I entertained the idea of creating two image layers, one with NIFeeds content and the other without. But that requires more fixups to the recipes (like disabling NIAuth in getty) and I didn't know what your BUILD_FROM_FEEDS patchset would look like.

I'm fine refactoring this after your patchset is in, but I think this is fine for the moment.

Also the `minimal-nilrt-*` content is the remnants of the `nilrt-nxg` pipeline. These changes only show as a diff because they had some convenient logic for building RAUC bundles. They are otherwise totally deprecated.